readers/common/src/main/java/com/criteo/hadoop/garmadon/reader/helper/ReaderUtils.java

```diff
@@ -0,0 +1,39 @@
+package com.criteo.hadoop.garmadon.reader.helper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ReaderUtils {
+    public static final Runnable EMPTY_ACTION = () -> {
+    };
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ReaderUtils.class);
+
+    protected ReaderUtils() {
+        throw new UnsupportedOperationException();
+    }
+
+    public static boolean retryAction(Runnable action, String exceptionStr, Runnable actionFailure) {
```

Comment:
[OPT] actionFailure could be Consumer<Throwable> to allow callee to do something with the exception